ctags/parsers/c.c

```diff
@@ -689,7 +689,7 @@ static const char *implementationString (const impType imp)
 
 #ifdef DEBUG
 
-#define boolString(c)   ((c) ? "TRUE" : "FALSE")
+#define boolString(c)   ((c) ? "true" : "false")
```

Comment:
this should probably be left out (although well, it's only used in debugging output, so doesn't matter so much)